tools/interop_matrix/client_matrix.py

```diff
@@ -258,5 +265,6 @@ def __init__(self, patch=[], runtime_subset=[], testcases_file=None):
         ('v1.16.0', ReleaseInfo(testcases_file='csharp__v1.3.9')),
         ('v1.17.1', ReleaseInfo(testcases_file='csharp__v1.3.9')),
         ('v1.18.0', ReleaseInfo()),
+        ('v1.19.0', ReleaseInfo()),
```

Comment:
The last 2 lines don't have the argument "testcases_file='csharp__v1.3.9'" . Just want to confirm it is okay and not an oversight. I vaguely remember there was some kind of explanation...

Reply:
Yes, this is as intended. Tests are passing fine.